pkg/app/rds_aurora_mysql.go

```diff
@@ -201,156 +212,77 @@ func (a *RDSAuroraMySQLDB) IsReady(context.Context) (bool, error) {
-}
+	pingCommand := []string{"sh", "-c", isReadyQuery}
 
-func (a *RDSAuroraMySQLDB) Insert(ctx context.Context) error {
-	db, err := a.openDBConnection()
+	_, stderr, err := a.execCommand(ctx, pingCommand)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "Error while Pinging the database: %s, app: %s", stderr, a.name)
 	}
-	defer func() {
-		if err = a.closeDBConnection(db); err != nil {
-			log.Print("Error closing DB connection", field.M{"app": a.name})
-		}
-	}()
 
-	query, err := db.Prepare("INSERT INTO pets VALUES (?,?,?,?,?,?);")
-	if err != nil {
-		return errors.Wrap(err, "Error preparing query")
-	}
+	log.Print("Ping to the application was success.", field.M{"app": a.name})
+	return nil
+}
 
-	// start a transaction
-	tx, err := db.Begin()
-	if err != nil {
-		return errors.Wrap(err, "Error beginning transaction")
-	}
+func (a *RDSAuroraMySQLDB) Insert(ctx context.Context) error {
+	log.Print("Adding entry to database", field.M{"app": a.name})
+	insertQuery := fmt.Sprintf(mysqlConnectionString+
+		"\"INSERT INTO pets VALUES ('Puffball', 'Diane', 'hamster', 'f', '1999-03-30', 'NULL');\"", a.host, a.username, a.password, a.dbName)
 
-	_, err = tx.Stmt(query).Exec("Puffball", "Diane", "hamster", "f", "1999-03-30", "NULL")
+	insertCommand := []string{"sh", "-c", insertQuery}
+	_, stderr, err := a.execCommand(ctx, insertCommand)
 	if err != nil {
-		return errors.Wrap(err, "Error inserting data into Aurora DB cluster")
-	}
-
-	if err = tx.Commit(); err != nil {
-		return errors.Wrap(err, "Error committing data into Aurora DB database")
+		return errors.Wrapf(err, "Error while inserting data into table: %s, app: %s", stderr, a.name)
 	}
-
+	log.Info().Print("Inserted a row in test db.", field.M{"app": a.name})
 	return nil
 }
 
-func (a *RDSAuroraMySQLDB) Count(context.Context) (int, error) {
-	db, err := a.openDBConnection()
+func (a *RDSAuroraMySQLDB) Count(ctx context.Context) (int, error) {
+	log.Print("Counting entries from database", field.M{"app": a.name})
+	countQuery := fmt.Sprintf(mysqlConnectionString+
+		"\"SELECT COUNT(*) FROM pets;\"", a.host, a.username, a.password, a.dbName)
+
+	countCommand := []string{"sh", "-c", countQuery}
+	stdout, stderr, err := a.execCommand(ctx, countCommand)
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrapf(err, "Error while counting data into table: %s, app: %s", stderr, a.name)
 	}
-	defer func() {
-		if err = a.closeDBConnection(db); err != nil {
-			log.Print("Error closing DB connection", field.M{"app": a.name})
-		}
-	}()
 
-	rows, err := db.Query("select * from pets;")
+	rowsReturned, err := strconv.Atoi(stdout)
 	if err != nil {
-		return 0, errors.Wrap(err, "Error preparing count query")
-	}
-	count := 0
-	for rows.Next() {
-		count++
+		return 0, errors.Wrapf(err, "Error while converting response of count query: %s, app: %s", stderr, a.name)
 	}
 
-	return count, nil
+	log.Info().Print("Counting rows in test db.", field.M{"app": a.name, "count": rowsReturned})
```

Comment:
```suggestion
	log.Info().Print("Counting rows in test db.", field.M{"app": a.name, "count": rowsReturned})
```

should this be, `Number of rows in test DB`? Or are we still counting the rows.